csrankings-j.csv

```diff
@@ -1085,6 +1085,7 @@ Jianzhong Li 0001,Harbin Institute of Technology,http://homepage.hit.edu.cn/jian
 Jianzhong Qi 0001,University of Melbourne,http://people.eng.unimelb.edu.au/jianzhongq,mxS6eHYAAAAJ
 Jianzhu Ma,Purdue University,https://www.cs.purdue.edu/homes/ma634,AATzYuAAAAAJ
 Jiaole Wang,Harbin Institute of Technology,http://faculty.hitsz.edu.cn/jlwang,utAk8ZMAAAAJ
+Jiaoyan Chen,University of Manchester,https://chenjiaoyan.github.io/,5Cy4z8wAAAAJ

```

Comment:
This is a disambiguation page. Please follow the instructions.

Reply:
Thanks. Just updated my homepage to the institute one.